### DISCUSSION TITLE
Nimble Logo

### DISCUSSION THREAD

**Discussion by timmwille** [Wakoma/nimble#76] [Ideas] [answered]
Earlier it was discussed here: https://github.com/Wakoma/nimble/issues/7

But at re:publica where we briefly met I had another Idea I wanted to express, let me know what you think!

Coming from this picture in mind:

![image](https://github.com/Wakoma/nimble/assets/8081055/07c21b2c-8151-45f6-8385-973e810a2478)

I extracted the following ideas for a Logo:
![image](https://github.com/Wakoma/nimble/assets/8081055/93829333-c2c9-4951-90a2-6b6f7c0c629d)

SVG export from drawio
![Nimble-Logo-idea drawio](https://github.com/Wakoma/nimble/assets/8081055/e73e4910-dc68-4359-8e47-5fc8c5015894)

**Comment by julianstirling** [comment]
I love the idea. I think we need it to look a little more artistic, so I took what you had and came up with:
![nimble](https://github.com/Wakoma/nimble/assets/12807051/3b3f59a4-24e0-4eb5-89c4-8374f47c51b5)

(editable inkscape svg):
![nimble](https://github.com/Wakoma/nimble/assets/12807051/bf86c410-c959-45a5-ae37-23325a004982)

@ericnitschke What do you think. If you like the general idea, someone more artistic than me or Timm could tart it up.

**Reply by timmwille** [reply]
interesting style, for me I like the technical approach so this above would identify a bit more with a gardening project or social/community approach. But it is still nice if you consider it a "nimble seed".

I wonder if a combination would work, as I understand the name should still be there, but would be nice to have a icon too?


![nimble-seed-technical](https://github.com/Wakoma/nimble/assets/8081055/eca001ad-157c-43d8-af85-026669e82dc5)

**Reply by timmwille** [reply]
![nimble-seed-technical-2](https://github.com/Wakoma/nimble/assets/8081055/6930ee8f-d539-44cf-ab43-3523351dc8fd)

**Reply by julianstirling** [reply]
I disagree that having a seed makes it gardening project, unless we think that the Apache software foundation are secretly a falconry project:
![image](https://github.com/Wakoma/nimble/assets/12807051/0f455c15-ce20-4873-bbd0-6866317ed396)

I think my issue with the Pappus into a network diagram is that looking at it now it just looks like a network diagram of a centralised star network topology; which is quite different from the message the project wants to give. I doubt anyone will see a dandelion pappus when they see the logo.

**Reply by timmwille** [reply]
Haha, I agree. Still my personal identifier, so it does not mean it might fit better for others. 

So I also had the thought about the centralized approach, then again we also do not talk about decentralisation but rather distribution of network nodes in form of a mesh. So my original idea was to have it as a "seeding node by node" approach.


Could also work like this, whatever the simplification would look like. 

![nimble-seed-technical-3](https://github.com/Wakoma/nimble/assets/8081055/ab552246-7eef-4fbd-9875-51ef014c5c8c)

**Reply by timmwille** [reply]
PS: this is what phind.com says in case anyone cares :laughing: → 


"The project name "Nimble - Seeding Node by Node" for an open-source mesh network node project suggests a focus on agility, efficiency, and the decentralized nature of mesh networks. The term "nimble" implies quickness and flexibility, which are desirable qualities in network technologies, especially those designed for dynamic environments like mesh networks. The phrase "seeding node by node" emphasizes the distributed approach of mesh networks, where each node contributes to the overall network structure and functionality. This could resonate well with developers and users interested in creating resilient, scalable, and self-healing networks.

Improving the project name could involve making it more descriptive of the project's core values or functionalities. For example, if the project aims to create highly reliable mesh networks that can operate independently of centralized infrastructure, incorporating terms like "resilient," "autonomous," or "self-sustaining" could enhance clarity. Additionally, emphasizing the project's open-source nature or its application in specific domains (e.g., disaster recovery, rural connectivity) could attract a more targeted audience.

As for implementing a whimsical seed as a symbol in the logo, it could serve as a metaphor for the project's mission to grow and expand the network organically, much like seeds sprouting and connecting to form a larger ecosystem. This visual element could effectively communicate the project's vision of a decentralized, interconnected network. The design should balance creativity with simplicity to ensure the logo remains accessible and memorable across various platforms and mediums."

**Reply by ericnitschke** [reply]
Despite my ongoing war against dandelions at home...I'm almost sold on the design having a single seed/pappus w. the word nimble.  Many connectivity-related projects have either wireless signals or some mesh typology, and it might be nice to have something a bit different. Designs somewhat related to what I'm envisioning from an ai tool: https://md.wakoma.net/eNOQppLETCeqFoKEzs0yGw#

**Reply by ericnitschke** [reply, marked as answer]
> I love the idea. I think we need it to look a little more artistic, so I took what you had and came up with: ![nimble](https://private-user-images.githubusercontent.com/12807051/336395846-3b3f59a4-24e0-4eb5-89c4-8374f47c51b5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDI0MjEwNzMsIm5iZiI6MTc0MjQyMDc3MywicGF0aCI6Ii8xMjgwNzA1MS8zMzYzOTU4NDYtM2IzZjU5YTQtMjRlMC00ZWI1LTg5YzQtODM3NGY0N2M1MWI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE5VDIxNDYxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM4OWU1YTk1YzdmYmMwMWQxZDRkODU2OTE5NGI5N2M3MjNmMGZhNmViZDNkZDM3MTNkNDhmMWU1MmI2MGM1YjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.aPUH27DXSYxJd0DyqVL_cZQUE7UAjmhfU62qXUQYWZM)
> 
> (editable inkscape svg): ![nimble](https://private-user-images.githubusercontent.com/12807051/336395961-bf86c410-c959-45a5-ae37-23325a004982.svg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDI0MjEwNzMsIm5iZiI6MTc0MjQyMDc3MywicGF0aCI6Ii8xMjgwNzA1MS8zMzYzOTU5NjEtYmY4NmM0MTAtYzk1OS00NWE1LWFlMzctMjMzMjVhMDA0OTgyLnN2Zz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE5VDIxNDYxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJkODgxY2UxN2NkOTJhYTY2NjQ5ZGE1ZGU1ZTQ2MzM4Mjk0OGJkM2VmZDA1ZTYyZWUxYmIwMjA3Nzc3MTNlZjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.PlexLiA29OZFUXbXoB7GnkrVvKRQbx-DDSmmVHvoDkw)
> 
> @ericnitschke What do you think. If you like the general idea, someone more artistic than me or Timm could tart it up.

This one's cool @julianstirling. Attempted tart up. 
![nimble logo 256](https://github.com/user-attachments/assets/f3b22f4f-4bcc-4e8d-bbe8-a4c1a027c6c4)

**Reply by ericnitschke** [reply]
> Haha, I agree. Still my personal identifier, so it does not mean it might fit better for others.
> 
> So I also had the thought about the centralized approach, then again we also do not talk about decentralisation but rather distribution of network nodes in form of a mesh. So my original idea was to have it as a "seeding node by node" approach.
> 
> Could also work like this, whatever the simplification would look like.
> 
> ![nimble-seed-technical-3](https://private-user-images.githubusercontent.com/8081055/339347553-ab552246-7eef-4fbd-9875-51ef014c5c8c.svg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDI0MjE4NjMsIm5iZiI6MTc0MjQyMTU2MywicGF0aCI6Ii84MDgxMDU1LzMzOTM0NzU1My1hYjU1MjI0Ni03ZWVmLTRmYmQtOTg3NS01MWVmMDE0YzVjOGMuc3ZnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDMxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAzMTlUMjE1OTIzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Y2JiYmY2MTA5ODZlZDgxMWM5YWJhMzJmYmZhYWJhYTk3Mjk1MWYzZDUzMjk3NDJmOTIzZDRmYjM4Yzc0M2I2MiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.-xI5eAvIKNyk2O0tazH99hD6_ADL5CL0BJ6FF-rUNBo)


![1](https://github.com/user-attachments/assets/2c4610ae-1880-4555-86e9-8e5843a159fb)
![2](https://github.com/user-attachments/assets/4a05fb40-b50b-4c71-a147-d9c77d5abf86)
![3](https://github.com/user-attachments/assets/bc55a975-23bc-473d-a16a-3cdbeab52efa)
credit: https://alexandraheller.com/centralized-decentralized-and-distributed/

**Comment by ericnitschke** [comment]
I really love this idea, thank you @timmwille @julianstirling. So much can be done with it. Is just the Pappus (https://en.wikipedia.org/wiki/Pappus_(botany)) enough in the logo? I suppose the 'clock' (https://en.wikipedia.org/wiki/Taraxacum_officinale) from the first pic can be shown on the website to drive the point home.

**Reply by julianstirling** [reply]
I'd think that anything more than the pappus would become quite a complex logo that is hard to use in multiple places. I do think that a nice image of the full "clock" starting to disperse is a nice image for the website. The image above is stunning, I assume it is not in the public domain? @timmwille do you have the reference/license for the above image?

**Reply by timmwille** [reply]
indeed I have used a public domain image, actually I downloaded it before pixabay introduced their strange license, here you can check the original and I also fund another one that should be public domain (cc0 is irrevocable) and might be even better.

https://pixabay.com/photos/flower-dandelion-seeds-wildflower-333643/

https://pixabay.com/photos/dandelion-seeds-dandelion-seeds-2266558/


for mine I just inverted the colors and made it bw.

**Comment by ericnitschke** [comment]
Well, it's been 2 years (https://github.com/Wakoma/nimble/issues/7), so probably about time to add a nimble logo...

Thinking this one should do for now. Courtesy of @timmwille & @julianstirling, with a few small changes.

<p align="center"><img alt="nimble logo 256" src="https://raw.githubusercontent.com/Wakoma/wakoma-assets/b84a00ba43119ff617c64942332dc7b5860567f4/nimble-assets/logos/nimble%20logo%20256.svg" height="auto" width="256"></p>

Any objections? Revisions always welcome!
